sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -150,9 +178,18 @@ export class AssessmentAESTController extends BaseController {
     @UserToken() userToken: IUserToken,
   ): Promise<PrimaryIdentifierAPIOutDTO> {
     try {
+      const currentApplicationId =
+        await this.applicationService.getApplicationIdByParentApplicationId(
+          applicationId,
+        );
+      if (!currentApplicationId) {
+        throw new NotFoundException(
+          `Current application for application ${applicationId} was not found.`,
+        );
+      }
       const manualAssessment =
         await this.studentAssessmentService.createManualReassessment(
-          applicationId,
+          currentApplicationId,
           payload.note,
           userToken.userId,
         );
```

Comment:
One more API which needs to adapt. 

![image](https://github.com/user-attachments/assets/ffd90ef3-5f1f-46b6-9c00-2488a9d8d31d)